addons/tidb/templates/componentdefinition-tidb.yaml

```diff
@@ -23,11 +37,10 @@ spec:
           - -c
           - |
             echo "start tidb..."
-            DOMAIN=$KB_NAMESPACE".svc{{ .Values.clusterDomain }}"
             exec /tidb-server --store=tikv \
             --advertise-address=${KB_POD_FQDN}{{ .Values.clusterDomain }} \
             --host=0.0.0.0 \
-            --path=${KB_CLUSTER_NAME}-tidb-pd.${DOMAIN}:2379 \
+            --path=${PD_ADDRESS}:2379 \
```

Comment:
it seems PD_ADDRESS already contains the port ?

Reply:
indeed